src/uct/ib/ud/accel/ud_mlx5.h

```diff
@@ -36,6 +36,7 @@ typedef struct {
     uct_ud_iface_t                      super;
     struct {
         uct_ib_mlx5_txwq_t              wq;
+        uct_ib_mlx5_mmio_mode_t         mmio_mode;
```

Comment:
instead of adding this, can we pass mmio_mode as arg to create_qp?

Reply:
the `mmio_mode` is known when creating IFACE

Reply:
and we create the qp when creating the iface..

Reply:
yes, we do it from `uct_iface_create()`, but from common UD iface code.
mmio_mode is a part of IB MLX5 config which can be accessed in UD_MLX5 iface create code